python/core/additions/qgsfunction.py

```diff
@@ -26,43 +24,52 @@
 from qgis._core import QgsExpressionFunction, QgsExpression, QgsMessageLog, QgsFeatureRequest, Qgis
 
 
-def register_function(function, arg_count, group, usesgeometry=False,
-                      referenced_columns=[QgsFeatureRequest.ALL_ATTRIBUTES], handlesnull=False, **kwargs):
+def register_function(
+    function,
+    group,
+    usesgeometry=False,
+    referenced_columns=[QgsFeatureRequest.ALL_ATTRIBUTES],
+    handlesnull=False,
+    **kwargs,
+):
     """
     Register a Python function to be used as a expression function.
 
-    Functions should take (values, feature, parent) as args:
-
-    Example:
-        def myfunc(values, feature, parent):
-            pass
+    The function signature may contains special parameters:
```

Comment:
Elsewhere the order is feature, parent, context. Would be nice to have it consistent even if that is purely cosmetics.